### ISSUE TITLE
AEM-9 produces false-positives

### ISSUE THREAD

**Issue by michalwronowski** [Cognifide/AEM-Rules-for-SonarQube#21]
When a session based object is accessed or returned within a private method that is called in constructor or `afterCreated` method it is still reported as an issue.

Below is an example of valid code that is marked as non compliant. Issue is reported for `getTags` method.
```
@SliceResource
public class Model implements InitializableModel {

	@Inject
	@RequestedPage
	private Page page;

	private List<Tag> tags;

	@Override
	public void afterCreated() {
		this.tags = getTags(page.getContentResource());
	}

	private List<Tag> getTags(Resource contentResource) {
		TagManager manager = contentResource.getResourceResolver().adaptTo(TagManager.class);
		return Arrays.asList(manager.getTags(contentResource));
	}
}
```

**Comment by mleczey** [comment]
That will be fixed before 0.5 release.

**Comment by mleczey** [comment]
This issue should be fixed by now. Could you confirm that?

**Comment by michalwronowski** [comment]
No more false-positives. Seems like fixed.
